src/packages/console/src/pages/createSitePage.tsx

```diff
@@ -143,99 +108,95 @@ export const CreateSitePage = (): React.ReactElement => {
-              </FormControl>
-              <TextField
-                variant='outlined'
-                margin='normal'
-                required
-                fullWidth
-                id='slug'
-                label='Site slug'
-                name='slug'
-                autoComplete='slug'
-                autoFocus
-                value={slug}
-                onChange={onSlugChanged}
-                error={slugError !== undefined}
-                helperText={slugError}
-              />
-              <Typography variant='subtitle2'>
-                {!slug ? 'This will be your everypage sub-domain e.g. hello.evrpg.com' : `Your everypage sub-domain will be ${slug}.evrpg.com`}
-              </Typography>
-              <TextField
-                variant='outlined'
-                margin='normal'
-                fullWidth
-                name='name'
-                label='Site name'
-                type='name'
-                id='name'
-                value={name}
-                onChange={onNameChanged}
-                error={nameError !== undefined}
-                helperText={nameError}
-              />
-              <TextField
-                variant='outlined'
-                margin='normal'
-                fullWidth
-                name='template'
-                label='Site template'
-                type='template'
-                id='template'
-                value={template ? template.name : 'Blank'}
-                onClick={onTemplateChoiceClicked}
-              />
-              <Button
-                type='submit'
-                fullWidth
-                variant='contained'
-                color='primary'
-                className={classes.submit}
-              >
-                Create Site
-              </Button>
-            </form>
-          )}
-        </Paper>
-      </Container>
-      {accounts && <AccountUpgradeDialog isOpen={isAccountUpgradePopupShowing} account={accounts.filter((account: Account): boolean => account.accountId === selectedAccountId).shift()} onCloseClicked={onAccountUpgradePopupCloseClicked} onUpgradeClicked={onAccountUpgradePopupUpgradeClicked} />}
-      <TemplateChooserModal isOpen={isTemplateChooserOpen} onChooseTemplateClicked={onChooseTemplateClicked} />
-    </div>
+    <ContainingView>
+      <ResponsiveContainingView size={12} sizeResponsive={{ small: 12, medium: 8, large: 5 }}>
+        <NavigationBar />
+        <Stack direction={Direction.Vertical} paddingVertical={PaddingSize.Wide2} isFullHeight={true}>
+          <Stack.Item growthFactor={1} shrinkFactor={1} />
+          <Box variant='card'>
+            <Stack direction={Direction.Vertical} shouldAddGutters={true}>
+              <Stack.Item alignment={Alignment.Center} gutterAfter={PaddingSize.Wide2}>
+                <Text tag='h1' variant='header5'>Create a new site</Text>
+              </Stack.Item>
+              {accounts === null ? (
+                <Text tag='p'>
```

Comment:
why p? also can you make it `variant={note}` so its a bit subtle. Also you can inline the text no need for the `{'...'}` on the next line